kinesis-cloud/start.sh

```diff
@@ -3,10 +3,14 @@
 # Source library
 . ../utils/helper.sh
 
+# Source Confluent Platform versions
+. ../utils/config.sh
```

Comment:
what do you thnk about sourcing `config.sh` inside `helper.sh` that way helper.sh gets this by default, then where we don't already source in helper we just source that in which will be standard across most of the scripts, instead of having to source in multiple scripts

Reply:
@rspurgeon while it requires an extra source, they are conceptually different things.  One is a library of functions, one is a configuration file.  I don't have a strong opinion to keep them as separate, but just wanted to highlight the reasoning for doing so.  WDYT?

Reply:
IMO the value of keeping them in a separate file is just so that tooling automation can update the file easily and prevent source control conflict issues.   I think the values in config.sh should be global to all our demos, so I don't see the need to require us to source in multiple script files when helper can just pull it in itself.  Or do it vice versa if it's the naming that's not right

Reply:
Addressed as you suggested.